bosh-deployment.yml

```diff
@@ -156,7 +156,6 @@ properties:
 
   blobstore:
     provider: s3
-    server_side_encryption: AES256
```

Comment:
No idea, not intentional. This is, however, consistent with my current error-ridden track record on even the tiniest change to any file. I will examine and fix -- thanks for catching it.